### PR TITLE
Cluster wording change

### PR DESCRIPTION
**Pull request by marians**
For https://github.com/giantswarm/giantswarm/issues/3991

To be merged together with https://github.com/giantswarm/docs-proxy/pull/27, which adds the redirects for the renamed guides.

Reviewers: Please check that the architecture diagrams are at least not worse than they were. I tried my best in a time-boxed effort with Puja.

### CODE REVIEW

File: src/content/guides/prepare-aws-account-for-tenant-clusters/index.md [hunk 1]
```diff
@@ -292,13 +292,13 @@ arn:aws:iam::<YOUR_ACCOUNT_ID>:role/GiantSwarmAdmin
 ## Configure the Giant Swarm organization {#configure-org}
 
 In the previous sections, we explained how to create two IAM roles in the
-AWS account that's going to run the Giant Swarm guest clusters.
+AWS account that's going to run the Giant Swarm tenant clusters.
 
-Giant Swarm guest clusters are owned by _organizations_, which allows to control
+Giant Swarm tenant clusters are owned by _organizations_, which allows to control
```
Comment (rossf7) [inline src/content/guides/prepare-aws-account-for-tenant-clusters/index.md]: allows you to

File: src/content/faq/index.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ The [Cron Job](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jo
...
+First we need IP space for our control plane and tenant clusters, meaning the underlying EC2 machines. We currently suggesting taking a Class C IP Block per tenant cluster, as this would allow some 240 machines or 120 if spread over two Availability Zones. While taking a full Class B IP Block would mean you can start well over 200 clusters, or e.g. 80 in three different Amazon Locations, you might be ok with using a smaller IP Range. These IPs, in case you somehow want IP to IP connections to those machines, need to be routable inside your old environment. Normally these can run over a NAT but still, the IPs should not be used inside your existing network in case systems inside the cluster need to talk to some of your old IPs, e.g. a database.
```
Comment (rossf7) [inline src/content/faq/index.md]: suggest taking

File: src/content/reference/giantswarm-onprem-architecture/index.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+Think of the Control Plane as the Giant Swarm team’s Kubernetes Cluster. The worker nodes are where your Tenant Clusters will end up on, controlled by our [kvm-operator](https://github.com/giantswarm/kvm-operator/) running inside the Control Plane. The above diagram shows our PoC setup in that keeps the Control Plane (including monitoring) on 3 machines. Based on amount and sizes of Tenant Clusters, we recommend having distinct monitoring machines once we go into production. We further recommend having at least 5 worker machines. Any additional machines will also be set up as worker nodes and made available for Tenant Clusters to be used by end users.
```
Comment (rossf7) [inline src/content/reference/giantswarm-onprem-architecture/index.md]: Based on the amount

File: src/content/reference/giantswarm-onprem-architecture/index.md [hunk 4]
```diff
@@ -1,41 +1,40 @@
...
+Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Web interface, you can start Tenant Clusters of different sizes. You choose the amount of vCores and RAM you want per Node of your Tenant Cluster and press “Create Cluster”. A few minutes later your cluster will be ready to use. It will be running on the Worker Nodes of the Control Plane, within KVM VMs networked together through a Flannel network. Each Tenant Cluster is separated from the other Tenant Clusters by a flannel VXLAN bridge. Inside of that bridge, the containers are networked with Calico BGP.
 
-Access to the K8s API goes through the Host Cluster. We can connected your Load Balancer for Ingress access. This Load Balancer can be either statically configured with healthchecks or controlled via API. Depending on the data center setup, there are different options that need to be evaluated.
+Access to the K8s API goes through the Control Plane. We can connected your Load Balancer for Ingress access. This Load Balancer can be either statically configured with healthchecks or controlled via API. Depending on the data center setup, there are different options that need to be evaluated.
```
Comment (rossf7) [inline src/content/reference/giantswarm-onprem-architecture/index.md]: We can connect

**Comment by marians** [comment]
Thanks for the extra scrutiny!

**Pull request merged**